### PR TITLE
HttpRequest usage

### PR DESCRIPTION
**Pull request by ivyazmitinov**

### CODE REVIEW

File: src/main/docs/guide/httpServer/requestResponse.adoc [hunk 1]
```diff
@@ -24,6 +24,15 @@ In fact, there are several higher level interfaces that can be bound to method p
 |`String hello(Cookies cookies)`
 |===
 
+IMPORTANT: Note, that api:io.micronaut.http.HttpRequest[] should be declared parametrized with concrete genereic type if request body is needed, e.g. ```java
```
Comment (saw303) [inline src/main/docs/guide/httpServer/requestResponse.adoc]:
```suggestion
IMPORTANT: Note, that api:io.micronaut.http.HttpRequest[] should be declared parametrized with concrete generic type if request body is needed, e.g. ```java
```

Comment (saw303) [inline src/main/docs/guide/httpServer/requestResponse.adoc]:
```suggestion
IMPORTANT: Note, that api:io.micronaut.http.HttpRequest[] should be declared parametrized with concrete generic type if request body is needed, e.g. ```java
```

Comment (saw303) [inline src/main/docs/guide/httpServer/requestResponse.adoc]:
```suggestion
IMPORTANT: Note, that api:io.micronaut.http.HttpRequest[] should be declared parametrized with concrete generic type if request body is needed, e.g. ```java
```

**Comment by jameskleeh** [comment]
@ivyazmitinov I don't have permissions to commit the typo fix that @saw303 added. Please allow me to commit to you branch (I think its an option when you submit the PR), or commit the change yourself

**Comment by ivyazmitinov** [comment]
@jameskleeh 
done

**Comment by jameskleeh** [comment]
@ivyazmitinov Thanks - can you rebase to 1.1.x since that is the earliest branch where this logic applies

**Comment by ivyazmitinov** [comment]
@jameskleeh 
done

**Pull request merged**